README.md

```diff
@@ -21,6 +21,9 @@ This open source plug-in uses Gradle as its build tool. [Install the latest vers
 
 ## Release Notes
 
+### Version 1.10
+- Removed admin user and administrative mode check.
```

Comment:
Bolster the explanation release notes with why.

ie. Feature is removed in UCD 6.2.X, so we've removed it in the Jenkins plugin too.